### ISSUE TITLE
Frame error on mojs.Timeline

### ISSUE THREAD

**Issue by gaou-piou** [legomushroom/mojs#147]
There is a frame error when mojs.Timeline repeats 

https://codepen.io/gaougalos/pen/eRqdoB

**Comment by legomushroom** [comment]
Hey @gaou-piou!

THanks for the issue, this is a bug indeed, I'll investigate shortly.

**Comment by xavierfoucrier** [comment]
You need to just create one shape and use the **isYoyo** property to do that animation. No? :thinking:

**Comment by legomushroom** [comment]
@xavierfoucrier I think `stagger` + `isYoyo` would work. You actually can do this in multiple ways, I usually prefer using the `property curves` and do it with one tween with no `appending`.

**Comment by gaou-piou** [comment]
You are right that animation can be written in various ways. I just made the animation to show the malfunction, not just to make it happen

**Comment by xavierfoucrier** [comment]
From my point of view, when I launched your animation, I immediately thought that the bug came from using the `append` method: thats why I proposed another way to do it. May be its a malfunction of the Timeline module: let's see @legomushroom investigation :wink:

**Comment by thewolf** [comment]
I think you can remove the animBallsDown and add 'stagger' to scaleX/scaleY:

`const balls = [].slice.call(document.querySelectorAll('.ball'))
const Balls = mojs.stagger(mojs.Html)
const animBallsUp = new Balls({
  quantifier: 3,
  el: balls,
  y: {0: -100},
  scaleY: {0.6: 1, duration: 'stagger(300, 50)'},
  scaleX: {1.4: 1, duration: 'stagger(300, 50)'},
  duration: 600,
  easing: 'quart.out',
  delay: 'stagger(0, 50)',
})

const anim = new mojs.Timeline({repeat: 999}).add(animBallsUp)

anim.play()`
